rllib/env/single_agent_env_runner.py

```diff
@@ -255,7 +255,7 @@ def _sample_timesteps(
                 # RLModule forward pass: Explore or not.
                 if explore:
                     to_env = self.module.forward_exploration(
-                        to_module, t=self.global_num_env_steps_sampled + ts
+                        to_module, t=self.global_num_env_steps_sampled
```

Comment:
Perfect! Thanks for fixing this logic. Now each EnvRunner is more robust in itself, keeping these up to date, at least until a new (and better) global count arrives.